CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish
+to make via issue, email, or any other method with the owners of this
+repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your
+interactions with the project.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the
+   layer when doing a build.
+2. Update the `README.md` and `CHANGELOG.md` with details of changes to the
+   interface, this includes new environment variables, exposed ports, useful
+   file locations and container parameters.
+3. Increase the version numbers in any examples files and the `README.md` to
+   the new version that this Pull Request would represent. The versioning
+   scheme we use is [SemVer](http://semver.org/).
```

Comment:
Instead, we should handle versioning. Our process instead is to update the CHANGELOG above by adding changes to the "Unreleased" category and following this convention https://keepachangelog.com/en/1.0.0/.

So no contributors should be updating versions of anything right now.

Reply:
I read this differently. I interpreted it as Bob submitted PR with documentation in `README.md` and his changes in `CHANGELOG.md`, but in the meantime we bumped the code version up to `x.y.z`, so now Bob is responsible for modifying his files to reflect this new version.

Maybe it can be written clearer.